languages/js/src/Oso.ts

```diff
@@ -131,7 +146,9 @@ export class Oso<
           return new Set(['*']);
         }
       }
-      actions.add(action);
+      // TODO(gj): do we need to handle the case where `action` is something
+      // other than a `Variable` or an `Action`? E.g., if it's an `Expression`?
```

Comment:
@gkaemmer any thoughts about this or the equivalent comment down below in `authorizedFields()`?

Reply:
Oh yeah I suppose the return type should be `Set<Action | Expression | "*">`